### ISSUE TITLE
Failing PLE 

### ISSUE THREAD

**Issue by konstjar** [OHDSI/Hydra#8]
Getting error with folder location in combination with cohortMethods lib in R

```
Generating outcomes
Warning: the condition has length > 1 and only the first element will be used
Generating outcomes for exposure NA and outcome NA
Warning: cannot open compressed file 'NA/betas.rds', probable reason 'No such file or directory'
Error in gzfile(file, "rb") : cannot open the connection
Calls: tryCatch ... <Anonymous> -> <Anonymous> -> lapply -> FUN -> readRDS -> gzfile
```

Attached Study in JSON format
[Graham replication.json.zip](https://github.com/OHDSI/Hydra/files/3559654/Graham.replication.json.zip)

**Comment by wivern** [comment]
@schuemie Could you please check this issue on the Truven dataset?

**Comment by schuemie** [comment]
Someone within JnJ has reported the same problem (on a different study). I'm trying to work with her to reproduce this problem, but so far no luck (it runs fine in my environment). I'll keep trying.

**Comment by schuemie** [comment]
Note that it's the positive control synthesis that is causing the problem, so if you're pressed for time you could try disabling that.

**Comment by schuemie** [comment]
I think I found the cause: there appear to not be enough negative controls with sufficient outcome count to fit the outcome models for the positive control synthesis. I've modified MethodEvaluation to give a meaningful error message in this case. Please try installing the latest version (from the develop branch). You'll also need to have the latest version of SqlRender:

```r
install.packages("SqlRender")
devtools::install_github("ohdsi/MethodEvaluation", ref="develop")
```

**Comment by konstjar** [comment]
![image](https://user-images.githubusercontent.com/4144173/64119690-b4978f80-cd9a-11e9-8395-059997f57afa.png)

It looks it works now. I'm getting this message

**Comment by schuemie** [comment]
So no negative control has enough outcomes during the time-at-risk to fit a model.

The only thing you can do is turn off positive control synthesis alltogether. Likely your entire study will prove to be underpowered though.

**Comment by pbr6cornell** [comment]
@schuemie , would this be a situation where we'd recommend doing CI calibration using on the negative controls?  And if so, where can we find the documentation to explain how to do CI calibration without positive control synthesis?

**Comment by schuemie** [comment]
Good point @pbr6cornell ! As Patrick mentioned, you can still perform CI calibration even without positive controls if you're willing to make the assumption systematic error does not change as a function of true effect size. The biggest challenge to that assumption would be bias towards the null, which arguably is less of a threat in a comparative cohort analysis. 

The instructions for doing this are [here](https://ohdsi.github.io/EmpiricalCalibration/articles/EmpiricalCiCalibrationVignette.html#fitting-the-empirical-null-and-specifying-an-assumption).
